text/0005-global-init.md

```diff
@@ -51,10 +51,12 @@ party libraries to use the global SDK before it is installed, which is
 addressed in a requirement stated below.
 
 The explicit initializer method should take independent `Tracer` and
-`Meter` objects (e.g., `opentelemetry.Init(Tracer, Meter)`).  The SDK
-may be installed no more than once.  After the first SDK installed,
-subsequent calls to the explicit initializer shall log console
-warnings.
+`Meter` factories (e.g., `opentelemetry.Init(TracerFactory,
+MeterFactory)`), factories because they facility explicitly named
+`Tracer` and `Meter` instances.  The SDK must not be installed more
+than once.  After the first SDK installed, subsequent calls to the
```

Comment:
While I agree that resetting the globals should be discouraged, completely restricting it feels unnecessary to me.

Reply:
How do you feel about the alleged risk? If anyone can install an SDK whenever they want, how do you (a main() function) prevent others from installing SDKs when you know what you want?

Reply:
my worry is how multi-host processes will deal with the global tracer? If one process hosts two apps, each of them maintainig it's dependency injection and creates own tracerfactory. For the libraries that needs to use "global" tracer, which do not use DI, second app can register itself somehow in the global tracer later.

It may be C# specific, but I can see how it also may be a case in Java.

Reply:
This OTEP is about how the globals are initialized, not about whether or not to support globals. I believe that debate has been had 100 times and globals are going to stay. That said, I'm trying to restrict them as much as possible by allowing them to be initialized once.  Applications that wish to use multiple SDKs should avoid the globals and use DI.  If there are libraries that use the globals, then there will be a problem in this scenario--either don't use globals or do use globals, they don't mix well.

Reply:
no, I'm not entering that debate =). In C# and .NET Core this problem is in many places. The best practice is to not use globals and use dependency injection (DI) instead. However there are still plenty of libraries that uses globals. So when you are registering yourself in DI, you also need to register in global.

The problem occurs when a single process is shared by two apps. Each of them use DI for all libraries that adopter DI. For libraries relying on static globals - ideally - should be a way to decide at call site which tracer to use. And each of those two apps can register themselves in that global tracer for awareness.

So in this situation there is a case of later update of a global tracer.

Reply:
Question is not about the debate, but about clearly documenting the need and reasons for supporting globals in OTel. 

Also, iirc, when we had this debate in the spring, we agreed that any documentation / facilities for globals should come with a disclaimer DO NOT USE THIS.